rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -53,6 +55,23 @@ SaveDialogBase::~SaveDialogBase()
 {
 }
 
+std::string SaveDataEntry::date() const
+{
+	return date_time::fmt_time("%c", mtime);
+}
+
+std::string SaveDataEntry::data_size() const
+{
+	std::string metric = "KB";
+	u64 sz = static_cast<u64>(std::ceil(size / 1000.0));
```

Comment:
utils::aligned_div is what you're looking for.

Reply:
done